spec/features/authentication/visitor_creates_an_account_spec.rb

```diff
@@ -1,43 +1,47 @@
 require "rails_helper"
 
-describe User, type: :feature do 
-  describe "A visitor" do 
-    it "can register as a user with correct field inputs" do 
+describe User, type: :feature do
+  describe "A visitor" do
+    it "can register as a user with correct field inputs" do
+      email = "#{(0..1000).to_a.sample}@email.com"
+      phone_number = "#{(0..1000).to_a.sample(10)}"
+
       visit root_path
       click_on "Create Account"
       expect(current_path).to eq(new_user_path)
 
-      fill_in "First Name", with: "First Name"
```

Comment:
@Sh1pley @riverswb It looks like this commit is *undoing* the styling commit that @njgheorghita committed previously.